metpy/plots/declarative.py

```diff
@@ -795,6 +807,8 @@ def draw(self):
         if self._need_redraw:
             if getattr(self, 'handle', None) is None:
                 self._build()
+            if self.colorbar is not None:
+                plt.colorbar(self.handle, orientation=self.colorbar, pad=0, aspect=50)
```

Comment:
We can avoid PyPlot here with:
```suggestion
                self.parent.ax.figure.colorbar(self.handle, orientation=self.colorbar, pad=0, aspect=50)
```

Reply:
Done.